bentoml/_internal/frameworks/mlflow.py

```diff
@@ -0,0 +1,213 @@
+from __future__ import annotations
+
+import os
+import typing as t
+import logging
+from typing import TYPE_CHECKING
+
+import bentoml
+from bentoml import Tag
+from bentoml.models import ModelContext
+from bentoml.exceptions import NotFound
+from bentoml.exceptions import BentoMLException
+
+from ..utils import LazyLoader
+
+if TYPE_CHECKING:
+    from bentoml.types import ModelSignature
+    from bentoml.types import ModelSignatureDict
+
+
+if TYPE_CHECKING:
+    import mlflow
+else:
+    mlflow = LazyLoader(
+        "mlflow",
+        globals(),
+        "mlflow",
+        exc_msg="`mlflow` is required in order to use module `bentoml.mlflow`, install "
+        "mlflow with `pip install mlflow`. For more information, refer to "
+        "https://mlflow.org/",
+    )
+
+
+MODULE_NAME = "bentoml.mlflow"
+MLFLOW_MODEL_FOLDER = "mlflow_model"
+API_VERSION = "v1"
+
+logger = logging.getLogger(__name__)
+
+
+def get(tag_like: str | Tag) -> bentoml.Model:
+    model = bentoml.models.get(tag_like)
+    if model.info.module not in (MODULE_NAME, __name__):
+        raise NotFound(
+            f"Model {model.tag} was saved with module {model.info.module}, not loading with {MODULE_NAME}."
+        )
+    return model
+
+
+def load_model(
+    bento_model: str | Tag | bentoml.Model,
+) -> mlflow.pyfunc.PyFuncModel:
+    """
+    Load the MLFlow model with the given tag from the local BentoML model store.
+
+    Args:
+        bento_model: Either the tag of the model to get from the store, or a BentoML
+            ``~bentoml.Model`` instance to load the model from.
+
+    Returns:
+        The MLFlow model loaded as PyFuncModel from the BentoML model store.
+
+    Example:
+
+    .. code-block:: python
+
+        import bentoml
+        pyfunc_model = bentoml.mlflow.load_model('my_model:latest')
+        pyfunc_model.predict( input_df )
+    """  # noqa
+    if not isinstance(bento_model, bentoml.Model):
+        bento_model = get(bento_model)
+
+    if bento_model.info.module not in (MODULE_NAME, __name__):
+        raise NotFound(
+            f"Model {bento_model.tag} was saved with module {bento_model.info.module}, not loading with {MODULE_NAME}."
+        )
+
+    return mlflow.pyfunc.load_model(bento_model.path_of(MLFLOW_MODEL_FOLDER))
+
+
+def import_model(
+    name: str,
+    model_uri: str,
+    *,
+    signatures: dict[str, ModelSignatureDict | ModelSignature] | None = None,
+    labels: dict[str, str] | None = None,
+    custom_objects: dict[str, t.Any] | None = None,
+    metadata: dict[str, t.Any] | None = None,
+    # ...
+) -> bentoml.Model:
+    """
+    Save a <FRAMEWORK> model instance to the BentoML model store.
```

Comment:
```suggestion
    import mlflow model from a URI to the BentoML model store.
```
I think you forgot to update the docstring here.